pkg/square/square_test.go

```diff
@@ -236,6 +236,77 @@ func TestSquareBlobPostions(t *testing.T) {
 	}
 }
 
+func TestTxShareRange(t *testing.T) {
+	type test struct {
+		name      string
+		txs       [][]byte
+		index     int
+		wantStart int
+		wantEnd   int
+		expectErr bool
+	}
+
+	txOne := types.Tx{0x1}
+	txTwo := types.Tx(bytes.Repeat([]byte{2}, 600))
+	txThree := types.Tx(bytes.Repeat([]byte{3}, 1000))
+
+	testCases := []test{
+		{
+			name:      "txOne occupies shares 0 to 0",
+			txs:       [][]byte{txOne},
+			index:     0,
+			wantStart: 0,
+			wantEnd:   1,
```

Comment:
if the transaction occupies only the first share, then the `wantEnd` should be equal to 0, since the range is inclusive, no?